lib/rubocop/config_loader.rb

```diff
@@ -220,6 +222,19 @@ def check_duplication(yaml_code, absolute_path)
         end
       end
 
+      def check_enabled_value(hash, parent = nil)
+        hash.each do |key, value|
+          check_enabled_value(value, key) if value.is_a?(Hash)
+
+          next unless key == 'Enabled' && value.class == String
```

Comment:
I think it'd be best to do this check for all common cop configurations that are booleans - e.g. `Safe`, `Autocorrect`, etc.

Reply:
Wow, It makes sense.

Reply:
I saw the docs https://docs.rubocop.org/en/latest/configuration/ .

I found only `Enabled` and `SafeAutoCorrect` to accept a boolean value key in available on .rubocop.yml.

Reply:
There's also `AutoCorrect` and `Safe`. Guess we forgot to document them.

Reply:
Ok, I'll add `AutoCorrect` and `Safe` to condition.